Stratis.Bitcoin.Common/Stratis.Bitcoin.Common.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netstandard1.6</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;1705;IDE0008;</NoWarn>
```

Comment:
I'll just leave this here. I think it is an issue in the sdk that will be fixed in version 2. https://github.com/dotnet/roslyn/issues/19640